api/dps/server.go

```diff
@@ -168,112 +168,108 @@ func (s *Server) GetRegisters(_ context.Context, req *GetRegistersRequest) (*Get
 
 // GetHeight implements the `GetHeight` function of the generated GRPC
 // server.
-func (s *Server) GetHeight(_ context.Context, req *GetHeightRequest) (*GetHeightResponse, error) {
+func (s *Server) GetHeightForBlock(_ context.Context, req *GetHeightForBlockRequest) (*GetHeightForBlockResponse, error) {
 	var blockID flow.Identifier
 	copy(blockID[:], req.BlockID)
 
-	height, err := s.index.Height(blockID)
+	height, err := s.index.HeightForBlock(blockID)
```

Comment:
Why context not passed to `HeightForBlock` (and `context.Background` used inside it instead)?

Reply:
`index.Reader` is an interface. It's implemented around the GRPC client, and around the database. It doesn't really make sense to propagate the context here.